internal/mesh/internal/controllers/sidecarproxy/builder/local_app.go

```diff
@@ -138,6 +138,7 @@ func destinationRulesByPort(allPorts []string, destinationRules []*pbauth.Destin
 	return out
 }
 
+//nolint:unparam
```

Comment:
Just curious but why was this necessary. AFAICT allPorts and dr are both used in the function.

Reply:
Or is the linter even smarter and figuring out that one of the two is always nil based on other code.

Reply:
I believe the latter - that was my best explanation for why it suddenly started failing on the linter upgrade